### PR TITLE
Add connected content to content cards overview

### PR DESCRIPTION
**Pull request by davidbielik**

### CODE REVIEW

File: _docs/_user_guide/message_building_by_channel/content_cards.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ guide_featured_list:
 ---
 
 {% alert note %}
-Braze recommends that customers who use our News Feed tool move over to our Content Cards messaging channel - it is more flexible, customizable, and reliable. It is also easier to find and use in the Braze product. Contact your Braze account manager for more information.
+Braze recommends that customers who use our News Feed tool move over to our Content Cards messaging channel - it is more flexible, customizable, and reliable. It is also easier to find and use in the Braze product. See our [Migration Guide][/docs/user_guide/message_building_by_channel/content_cards/migrating_from_news_feed/] or contact your Braze account manager for more information.
```
Comment (KellieHawks) [inline _docs/_user_guide/message_building_by_channel/content_cards.md:31]: Please change link to:
(https://www.braze.com/docs/user_guide/message_building_by_channel/content_cards/migrating_from_news_feed/)

Comment (davidbielik) [reply _docs/_user_guide/message_building_by_channel/content_cards.md:31]: do we prefer to use relative links? it'll make it work when clicking in a local environment

Comment (KellieHawks) [reply _docs/_user_guide/message_building_by_channel/content_cards.md:31]: True, normally I would suggest including {{site.baseurl}} but inside alerts the format has to be this way or else the link doesn't load
Screen Shot 2020-05-07 at 11 00 24 AM

Comment (davidbielik) [reply _docs/_user_guide/message_building_by_channel/content_cards.md]: image
fixed! my mistake with the square brackets

**Pull request merged**